src/test/kotlin/com/exactpro/th2/common/event/TestEvent.kt

```diff
@@ -36,6 +39,7 @@ import java.time.LocalDate
 import java.time.LocalDateTime
 import java.time.LocalTime
 import java.time.ZoneOffset
+import kotlin.IllegalArgumentException as IllegalArgumentException1
```

Comment:
Why do you need that? Kotlin's IllegalArgumentException is the alias to Java's IllegalArgumentException. It looks like we are doing something wrong here